### PR TITLE
Move body overflow from inline style to CSS

### LINKED ISSUE #22: handle body overflow property modification via css classes
line 279
line 290

### PR DESCRIPTION
**Pull request by Friedrich-Mocker**

### CODE REVIEW

**Comment by ain** [comment]
For the reference, please also see http://conventions.interactive-pioneers.com/VCS#commits

File: src/iptools-jquery-modal.js [hunk 1]
```diff
@@ -32,7 +32,9 @@
         closeButton: '__button-close',
         spinner: '__spinner',
         content: '__content'
-      }
+      },
+      overflowHidden: 'body-overflow-hidden'
+
```
Comment (GeorgMeyer23) [inline src/iptools-jquery-modal.js]: remove empty line

File: src/iptools-jquery-modal.js [hunk 2]
```diff
@@ -57,7 +59,9 @@
       spinnerHTML: '',
       width: '80%',
       height: 'auto',
-      zIndex: 102
+      zIndex: 102,
+      overflowHiddenClass: classes.overflowHidden,
+      overflowAutoClass: classes.overflowAuto,
```
Comment (GeorgMeyer23) [inline src/iptools-jquery-modal.js]: remove obsolete overflowAutoClass

File: src/iptools-jquery-modal.scss [hunk 3]
```diff
@@ -141,6 +141,10 @@ $IPTModal_animation-speed: 0.25s;
   }
 
 }
+/* Body modifier classes */
```
Comment (GeorgMeyer23) [inline src/iptools-jquery-modal.scss]: add empty line

**Comment by ain** [comment]
`README` update is required as well.

**Comment by ain** [comment]
Rebased to #69. Closing.

**Pull request closed without merging**